Lesson_23/generator.py

```diff
@@ -0,0 +1,26 @@
+# import nltk
+
+# Loading the word list
+# nltk.download('words')
+
+import random
+from nltk.corpus import words
+
+# Download a big list of words
```

Comment:
Words are not uniq

print(len(list(random_unique_words(word_count))))

print(len(set(random_unique_words(word_count))))